charts/pgadmin4/templates/deployment.yaml

```diff
@@ -111,10 +111,11 @@ spec:
                 secretKeyRef:
           {{- if not .Values.existingSecret }}
                   name: {{ $fullName }}
+                  key: password
           {{- else }}
                   name: {{ .Values.existingSecret }}
-          {{- end }}
-                  key: password
+                  key: {{ .Values.secretKeys.pgadminPasswordKey }}
+          {{- end }}                  
```

Comment:
Can you remove the trailing spaces?

Reply:
I'm out of my workplace. I'll do it later in a week or so.

Reply:
@rowanruseler done